src/components/notifications/static/static.spec.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render } from 'react-testing-library';
```

Comment:
Although `react-testing-library` works fine in this case, I'd welcome if we could use `test-utils` (from `src/test-utils`). This will prepare the tests to be expanded (e.g. for different locales). And it will set a good example of using `test-utils`.